lib/Statocles.pm

```diff
@@ -155,8 +171,8 @@ sub startup {
     $app->plugin( Export => );
     push @{$app->export->pages}, '/sitemap.xml', '/robots.txt';
     $app->plugin( AutoReload => );
-
-    if ( my $theme = $app->config->{theme} ) {
+    do {
+        my $theme = $app->config->{theme} //= '+Statocles/theme/default';
```

Comment:
This helped with the unit tests, if it's a clue to why the tests are broken

Reply:
Hmm... Might be that the `config_override` in Test::Mojo doesn't work the way I assumed it did (which, I don't know why I assumed it would merge with the defaults, because the docs explicitly say it does not)...

Reply:
It merges with the default when making a `statocles.conf`, which could be added to each each `share` directory. Providing a hash to Test::Mojo makes a new config.